### DISCUSSION TITLE
Remote Printing application

### DISCUSSION THREAD

**Discussion by SComps** [lenticularis39/axpbox#93] [General]
I've started working on a remote "printer" that can be used with axpbox.

If you're interested in the project, I'd love to hear from you.  It's written in VB.NET for Windows, and in it's current state seems to work fairly well.  I'm sure there's a lot more to do with it, so any input would be appreciated.

Thanks to everyone involved with axpbox, and the emulator community in general.

[https://github.com/SComps/RemoteLPT]

**Comment by lenticularis39** [comment]
I'm not that much interested in Windows software, since I don't have a permanent Windows machine, I prefer not to use Windows when I don't have to :D

**Comment by fsword7** [comment]
There is Virtual 1403 remote printer for all platforms and produces PDF printout files with green-bar background image.  That emulates IBM 1403 line printer.  It is available on this GitHub.   VB.NET is for Windows only.   Go language is available for all platforms (Window, Linux and MacOS).  Virtual 1403 is written in Go language.

**Reply by SComps** [reply]
I've seen the 1403.  Really slick to be fair.  As I mentioned, this is more a personal project that I thought I'd share.  Maybe for grins I'll see how much fun it'll be to compile under mono as well.
